src/widget/wtracktableview.h

```diff
@@ -69,6 +69,10 @@ class WTrackTableView : public WLibraryTableView {
     bool slotRestoreCurrentViewState() {
         return restoreCurrentViewState();
     };
+    void slotSelectTrack(const TrackId& trackId) {
```

Comment:
Please move the implementation into the .cpp file. I suggest to define only trivial, single-line functions inline.